com.endlessm.EknServices.json.in

```diff
@@ -2,7 +2,7 @@
     "app-id": "com.endlessm.EknServices2",
     "branch": "master",
     "runtime": "com.endlessm.apps.Platform",
-    "runtime-version": "master",
+    "runtime-version": "3",
```

Comment:
We should keep this as `master` in the master git branch

Reply:
Argh! Sorry, leftover change.